React/Views/RCTView.m

```diff
@@ -1519,6 +1519,9 @@ - (void)setMouseDownCanMoveWindow:(BOOL)mouseDownCanMoveWindow{
 	_mouseDownCanMoveWindow = mouseDownCanMoveWindow;
 }
 
+- (BOOL)allowsVibrancy {
+  return _allowsVibrancyInternal;
+}
```

Comment:
@anandrajeswaran I think this is the preferred way to set this otherwise readonly NSView property? The alternative was to override the property in RCTView to be `assign` but that seemed wrong?